src/site/xdoc/server/config-elasticsearch.xml

```diff
@@ -200,6 +189,49 @@
 
     </section>
 
+    <section name="SSL Trusting Configuration">
+
+        <p>
+            In some cases, you want to secure ES to protect it from unauthorized requests,
+            assuming with the ES is using <strong>https</strong> with a self signed certificate.
+            Which means you should configure the ES RestHighLevelClient to trust your self signed certificate.
+
+            There are two ways on client side: ignoring SSL check or configure to trust the server's certificate.
+            In case you want to ignore the SSL check, simply, just don't specify below options. Otherwise, configuring the trust
```

Comment:
I thing we get something wrong here. Insecure by default is convenient, but is also the breach for mis-configurations.

We should:
 - Rely on JVM certificate by default
 - Have an option to effectively disable trust (that defaults to **false**
 - Have a way to load the trustStore...

Reply:
Right, thanks!